olive/model.py

```diff
@@ -163,47 +166,16 @@ def load_model(self) -> onnx.ModelProto:
         return onnx.load(self.model_path)
```

Comment:
>return onnx.load(self.model_path) [](http://example.com/codeflow?start=8&length=33)

two things:
1. should we cache the loaded model like PyTorch model does?
2. what about the external data scenarios?

Reply:
As discussed offline, `get_io_config` now calls `onnx.load` directly with `use_external_data=False`. 
For `model.load_model`, we can think about this later if needed.